nautobot_ssot/jobs/base.py

```diff
@@ -147,6 +252,13 @@ def __init__(self):
         self.sync = None
         self.kwargs = {}
         self.commit = False
+        self.diff = None
+        self.source_adapter = None
+        self.target_adapter = None
+        # Default diffsync flags. You can overwrite them at any time.
+        self.diffsync_flags = (
+            DiffSyncFlags.CONTINUE_ON_FAILURE | DiffSyncFlags.LOG_UNCHANGED_RECORDS | DiffSyncFlags.SKIP_UNMATCHED_DST
```

Comment:
I don't think we should default `SKIP_UNMATCHED_DST` as a flag - that should be opt-in.